pa11y_scripts/.crt-view.json

```diff
@@ -1,19 +1,20 @@
 {
-    "urls": [
-        {
-          "useIncognitoBrowserContext": true,
-          "url": "http://127.0.0.1:8000/form/view/",
-          "actions": [
-            "wait for url to be http://127.0.0.1:8000/accounts/login/?next=/form/view/",
-            "set field #id_username to pa11y_tester",
-            "set field #id_password to imposing40Karl5monomial",
-            "click #login",
-            "wait for url to be http://127.0.0.1:8000/form/view/?status=new&status=open&no_status=false",
-            "wait for element #id_status to be visible",
-            "click element #submitted-sort",
-            "wait for url to be http://127.0.0.1:8000/form/view/?sort=create_date&status=new&status=open"
-          ],
-          "screenCapture": "pa11y-screenshots/crt-sort-view.png"
-        }
-    ]
+  "urls": [
+    {
+      "useIncognitoBrowserContext": true,
+      "url": "http://127.0.0.1:8000/form/view/",
+      "actions": [
+        "wait for url to be http://127.0.0.1:8000/accounts/login/?next=/form/view/",
+        "set field #id_username to pa11y_tester",
+        "set field #id_password to imposing40Karl5monomial",
+        "click #login",
+        "wait for path to be /form/view/",
+        "screen capture pa11y-screenshots/crt-view-step1.png",
```

Comment:
we can remove the screen capture here

Reply:
pa11y is timing out without screen capture. i'm going to leave it there.